.github/workflows/release_prep.sh

```diff
@@ -53,6 +53,13 @@ Add to your \`MODULE.bazel\` file:
 
 \`\`\`starlark
 bazel_dep(name = "aspect_rules_py", version = "${TAG:1}")
+
+EOF
+
+# Collect the tools snippet from the use_release module file.
+awk 'f;/--SNIP--/{f=1}' e2e/use_release/MODULE.bazel
```

Comment:
I expected this would just take the snippet from the use_release e2e INSTEAD of the smoke e2e, as each one should be fully self-contained there's no reason we have to compose more than one of them

Reply:
The `use_release` didn't have any of the Python toolchain stuff in. I don't know if it needs it, although, it's working.

Reply:
That sounds like a bug in `use_release` then. If it's not complete then it doesn't work. I'll TAL

Reply:
Yup it's already red, I missed adding it to the test matrix https://github.com/aspect-build/rules_py/blob/main/.github/workflows/ci.yaml#L23